bitcoin/src/blockdata/witness.rs

```diff
@@ -246,29 +289,70 @@ impl Witness {
         if self.witness_elements <= 1 {
```

Comment:
We should deprecate `second_to_last` now. If getting arbitrary elements from the back is useful we should have `nth_rev` method for that.